### PR TITLE
Update RocksDB to 6.5:

### PR DESCRIPTION
**Pull request by mellery451**
* update EP and find package requirements
* minor protobuf/libarchive build fixes
* change travis release builds to nounity to ameliorate vm memory exhaustion.

FIXES: #3223, #3232

### CODE REVIEW

**Comment by mellery451** [comment]
@movitto I've also added a build fix here for fedora systems - any feedback appreciated

**Comment by movitto** [comment]
@mellery451 gave it a test locally and yes, changing **lib** to **${CMAKE_INSTALL_LIBDIR}** in Protobuf.cmake seemed to have the intended effect, the Fedora build now succeeds. Please note I didn't dive into the other changes extensively though.

File: Builds/CMake/deps/Findlibarchive_pc.cmake [hunk 1]
```diff
@@ -20,3 +20,5 @@ find_path (LIBARCHIVE_INCLUDE_DIR
     ${libarchive_PC_INCLUDEDIR}
     ${libarchive_PC_INCLUDEDIRS}
   NO_DEFAULT_PATH)
+
```
Comment (movitto) [inline Builds/CMake/deps/Findlibarchive_pc.cmake]: Some extraneous whitespace additions that you may want to exclude before the final patch is merged (also at the end of FindRocksDB.cmake above)

Comment (mellery451) [reply Builds/CMake/deps/Findlibarchive_pc.cmake]: cleaned those trailing blanks - thanks

**Pull request closed without merging**